sources/assets/Stride.Core.Assets/Compiler/PackageCompiler.cs

```diff
@@ -21,8 +19,12 @@ static PackageCompiler()
         {
             // Compute StrideSdkDir from this assembly
             // TODO Move this code to a reusable method
+
+            var isUnix = Environment.OSVersion.Platform == PlatformID.Unix;
             var codeBase = typeof(PackageCompiler).Assembly.Location;
-            var uri = new UriBuilder(codeBase);
+            // Uri paths starts with // on Unix systems 
+            codeBase = isUnix ? "/" + codeBase : codeBase;
+            var uri = new UriBuilder(new Uri(codeBase));
```

Comment:
Absolute paths start with a `/`. Relative paths don't. Are we sure about this change?

Instead of the constant, I would rather use the `Path.DirectorySeparatorChar` static field.

Edit: why are we even using a Uri here since we are only interested with a file path?